pkg/aws/s3store_test.go

```diff
@@ -0,0 +1,98 @@
+package aws_test
+
+import (
+	"bytes"
+	"encoding/hex"
+	"net/url"
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/storacha/go-libstoracha/ipnipublisher/store"
+	"github.com/storacha/go-libstoracha/testutil"
+	paws "github.com/storacha/piri/pkg/aws"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/minio"
+)
+
+func TestS3StoreReplace(t *testing.T) {
+	if os.Getenv("CI") != "" && runtime.GOOS != "linux" {
+		t.SkipNow()
+	}
```

Comment:
I am getting failures on OSX locally when running this test, was the intention for this to only run when both CI is set and the OS is Linux?